sktime/classification/frequency_based/_rise.py

```diff
@@ -133,118 +203,128 @@ def fit(self, X, y):
             self.intervals[i][1] = rng.randint(
                 self.intervals[i][0] + self.min_interval, self.series_length
             )
+
         # Check lag against global properties
         self.acf_lag_ = self.acf_lag
         if self.acf_lag > self.series_length - self.acf_min_values:
             self.acf_lag_ = self.series_length - self.acf_min_values
         if self.acf_lag < 0:
             self.acf_lag_ = 1
         self.lags = np.zeros(self.n_estimators, dtype=int)
-        for i in range(0, self.n_estimators):
-            temp_lag = self.acf_lag_
-            if (
-                temp_lag
-                > self.intervals[i][1] - self.intervals[i][0] - self.acf_min_values
-            ):
-                temp_lag = (
-                    self.intervals[i][1] - self.intervals[i][0] - self.acf_min_values
-                )
-            if temp_lag < 0:
-                temp_lag = 1
-            self.lags[i] = int(temp_lag)
-            acf_x = np.empty(shape=(n_instances, self.lags[i]))
-            ps_len = (self.intervals[i][1] - self.intervals[i][0]) / 2
-            ps_x = np.empty(shape=(n_instances, int(ps_len)))
-            for j in range(0, n_instances):
-                acf_x[j] = acf(
-                    X[j, self.intervals[i][0] : self.intervals[i][1]], temp_lag
-                )
-                ps_x[j] = ps(X[j, self.intervals[i][0] : self.intervals[i][1]])
-            transformed_x = np.concatenate((acf_x, ps_x), axis=1)
-            #            transformed_x=acf_x
-            tree = clone(self.base_estimator)
-            # set random state, but not the same, so that estimators vary
-            tree.set_params(**{"random_state": rng.randint(np.iinfo(np.int32).max)})
-            tree.fit(transformed_x, y)
+
+        trees = [
+            self._make_estimator(
+                append=False, random_state=rng.randint(np.iinfo(np.int32).max)
+            )
+            for i in range(self.n_estimators)
+        ]
+
+        # Parallel loop
+        worker_rets = Parallel(n_jobs=self.n_jobs)(
+            delayed(_parallel_build_trees)(
+                X,
+                y,
+                t,
```

Comment:
Please replace `t` with `tree` - much easier to understand with more meaningful variable names :)